persistence/remote/src/main/java/org/infinispan/persistence/remote/configuration/Attribute.java

```diff
@@ -45,6 +46,7 @@ public enum Attribute {
    PORT("port"),
    PROTOCOL("protocol"),
    PROTOCOL_VERSION("protocol-version"),
+   @Deprecated(forRemoval = true, since = "12.1")
    RAW_VALUES("raw-values"),
```

Comment:
Why not remove the attributes?

Reply:
We still need to warn about their removal when parsing older schemas

Reply:
So, when can we remove them? It looks like we have to keep them forever.

Reply:
When we stop parsing older schema versions. 
We should probably define a schedule for that

Reply:
Definitely.